ckanext/canada/i18n/fr/LC_MESSAGES/canada.po

```diff
@@ -1547,8 +1547,8 @@ msgstr "http://ouvert.canada.ca/fr/formulaire/faites-nous-part-de-vos-commentair
 msgid "Open Government Log In"
 msgstr "Ouverture de session Gouvernement ouvert"
 
-msgid "/en/user"
-msgstr "/fr/user"
```

Comment:
we still need the /en/user "translation" for the places we use it on the public site